static/js/app.js

```diff
@@ -4,16 +4,17 @@ $(function () {
             "": "navigateEmpty",
             "/?marker=:id&start_date=:start&end_date=:end&show_fatal=:showFatal&show_severe=:showSevere&show_light=:showLight&show_inaccurate=:showInaccurate&zoom=:zoom&lat=:lat&lon=:lon": "navigate"
         },
-        navigate: function (id, start, end, showFatal, showSevere, showLight, showInaccurate, zoom, lat, lon) {
-            app.model.set("currentMarker", parseInt(id));
-            app.model.set("dateRange", [new Date(start), new Date(end)]);
-            app.model.set("showFatal", showFatal);
-            app.model.set("showSevere", showSevere);
-            app.model.set("showLight", showLight);
-            app.model.set("showInaccurateMarkers", showInaccurate);
-            app.map.setZoom(parseInt(zoom));
-            app.map.setCenter(new google.maps.LatLng(lat, lon));
-        },
+        //removed navigate function because we don't use backbone navigation for now
```

Comment:
Why not use Backbone navigation? It means that the page URL always stays just the root URL of the app, without the parameters. Right now (i.e. before this PR) we do get the URL parameters updated whenever we move the map (see http://anyway-unstable.herokuapp.com/), thanks to Backbone navigation. Perhaps it's worth consulting with UX experts before removing this functionality (and being left with the only option to copy the full URL being through the "copy link" button).